examples/camera/first_person_view_model.rs

```diff
@@ -0,0 +1,250 @@
+//!
+//! | Key Binding          | Action        |
+//! |:---------------------|:--------------|
+//! | mouse                | Look around   |
+//! | arrow up             | Decrease FOV  |
+//! | arrow down           | Increase FOV  |
+
+use bevy::color::palettes::tailwind;
+use bevy::input::mouse::MouseMotion;
+use bevy::pbr::NotShadowCaster;
+use bevy::prelude::*;
+use bevy::render::view::RenderLayers;
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .add_systems(
+            Startup,
+            (
+                spawn_view_model,
+                spawn_world_model,
+                spawn_lights,
+                spawn_text,
+            ),
+        )
+        .add_systems(Update, (move_player, change_fov))
+        .run();
+}
+
+// Components.
+
+#[derive(Debug, Component)]
+struct Player;
+
+#[derive(Debug, Component)]
+struct WorldModelCamera;
+
+// Setup systems.
+
+fn spawn_view_model(
+    mut commands: Commands,
+    mut meshes: ResMut<Assets<Mesh>>,
+    mut materials: ResMut<Assets<StandardMaterial>>,
+) {
+    let arm = meshes.add(Cuboid::new(0.1, 0.1, 0.5));
+    let arm_material = materials.add(Color::from(tailwind::TEAL_200));
+
+    commands
+        .spawn((
+            Name::new("Player"),
+            SpatialBundle {
+                transform: Transform::from_xyz(0.0, 1.0, 0.0),
+                ..default()
+            },
+            Player,
+        ))
+        .with_children(|parent| {
+            parent.spawn((
+                Name::new("World Model Camera"),
+                Camera3dBundle {
+                    projection: PerspectiveProjection {
+                        fov: 90.0_f32.to_radians(),
+                        ..default()
+                    }
+                    .into(),
+                    ..default()
+                },
+                WorldModelCamera,
+            ));
+            parent.spawn((
+                Name::new("View Model Camera"),
+                Camera3dBundle {
+                    camera: Camera {
+                        // Bump the order to render on top of the world model.
+                        order: 1,
+                        ..default()
+                    },
+                    projection: PerspectiveProjection {
+                        fov: 70.0_f32.to_radians(),
+                        ..default()
+                    }
+                    .into(),
+                    ..default()
+                },
+                // Only render objects on layer 1.
+                RenderLayers::layer(1),
+            ));
+
+            parent.spawn((
+                Name::new("Arm"),
+                MaterialMeshBundle {
+                    mesh: arm,
+                    material: arm_material,
+                    transform: Transform::from_xyz(0.2, -0.1, -0.25),
+                    ..default()
+                },
+                // Ensure the arm is only rendered by the view model camera.
+                RenderLayers::layer(1),
```

Comment:
Maybe make this a constant and use it here and in the camera definition?

```rust
const VIEW_MODEL_LAYER: RenderLayers = RenderLayers::layer(0);
```

Reply:
I was thinking about that, but then it would seem weird to me that the world model does not have a constant since layer 0 is the default. What do you think?

Reply:
I changed my opinion: [`a7e69b6` (#13828)](https://github.com/bevyengine/bevy/pull/13828/commits/a7e69b68bd9f37e989b810ac97bdb21c535422c9)
I tried out your suggestion and it looks better this way. Thanks!